src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -621,7 +641,11 @@ type FrameworkIdentifier =
         | DotNetFramework FrameworkVersion.V4_5_2 -> [ DotNetFramework FrameworkVersion.V4_5_1; DotNetStandard DotNetStandardVersion.V1_2 ]
         | DotNetFramework FrameworkVersion.V4_5_3 -> [ DotNetFramework FrameworkVersion.V4_5_2; DotNetStandard DotNetStandardVersion.V1_2 ]
         | DotNetFramework FrameworkVersion.V4_6 -> [ DotNetFramework FrameworkVersion.V4_5_3; DotNetStandard DotNetStandardVersion.V1_3 ]
-        | DotNetFramework FrameworkVersion.V4_6_1 -> [ DotNetFramework FrameworkVersion.V4_6; DotNetStandard DotNetStandardVersion.V1_4 ]
+        | DotNetFramework FrameworkVersion.V4_6_1 ->
+            match DotnetToolingSupport.mode with
+            | ToolingSupportMode.NativeSupport -> [ DotNetFramework FrameworkVersion.V4_6; DotNetStandard DotNetStandardVersion.V1_4 ]
+            // .NET Standard 2.0 will propagate "down", so we don't need any switches on the subsequent frameworks
```

Comment:
Just note that depending on this will subtly prefer netstandard1.5 in some situations over netstandard2.0 (for example in net462).
The possible impact of this is smaller as expected as we forward all packages to nuget in the final step, but the list of packages might differ.